cylc/flow/cfgspec/suite.py

```diff
@@ -189,6 +189,9 @@
             tasks manually near the end of a suite run, during suite
             development and debugging.
         ''')
+        Conf('abort if any task fails', VDR.V_BOOLEAN, desc='''
+            TODO: Remove this at Cylc9
```

Comment:
Remove this TODO, this text gets rendered in the docs and all deprecated/obsolete configs will get removed at Cylc9.